src/amber/cli/commands/exec.cr

```diff
@@ -30,14 +30,23 @@ module Amber::CLI
         system("cp #{_filename} #{@filename}") if _filename
       end
 
+      def code(args)
+        <<-CRYSTAL
+        result = begin
+          #{args.code}
```

Comment:
If we want to maintain functionality as before (for symbols), this may need string concatenation instead of interpolation. I'm not totally sure that will fix it though.